hollow/src/main/java/com/netflix/hollow/api/consumer/fs/HollowFilesystemBlobRetriever.java

```diff
@@ -223,12 +226,17 @@ public InputStream getInputStream() throws IOException {
         @Override
         public InputStream getInputStream() throws IOException {
 
-            Path tempPath = path.resolveSibling(path.getName(path.getNameCount()-1) + "-" + UUID.randomUUID().toString());
+            Path tempPath = path.resolveSibling(PARTIAL_DOWNLOAD + path.getName(path.getNameCount()-1));
```

Comment:
nice, although FWIW most  partial download files I've seen end in `.part` or similar (rather than being prefixed)

Reply:
There is some fragile filename parsing at other places in code, and it seemed to work better to have partial in the prefix as opposed to suffix. One eg. is checking if filename startswith "snapshot.".